src/menu/stateful-container.js

```diff
@@ -149,11 +154,65 @@ export default class MenuStatefulContainer extends React.Component<
         }
         this.handleEnterKey(event);
         break;
+      default:
+        if (this.props.typeAhead) this.handleAlphaDown(event);
+        break;
+    }
+  };
+  handleAlphaDown = (event: KeyboardEvent) => {
+    const rootRef = this.props.rootRef ? this.props.rootRef : this.rootRef;
+    const prevIndex = this.state.highlightedIndex;
+
+    this.typeAheadChars += event.key;
+    this.typeAheadTimeOut = setTimeout(() => {
+      this.typeAheadChars = '';
+    }, 500);
+
+    var nextIndex = prevIndex;
+    var list = this.getItems();
+    var notMatch = true;
+    for (let n = 0; n < list.length; n++) {
+      let label = 'label' in list[n] ? list[n].label : list[n].id;
```

Comment:
We should remove `id` here and in the other loop below. All of the menu elements have a randomly generated id

Reply:
Yeah, thanks! I will modify it.